etc/testing/spout/go.mod

```diff
@@ -0,0 +1,3 @@
+module etc/testing/spout/main
```

Comment:
`github.com/pachyderm/pachyderm/v2/etc/testing/spout/main` ?

I'm not sure what the rules are for nested modules; they are definitely allowed, but I'm unsure of the semantics.

Reply:
I am not entirely sure what you mean here. This was generated by go mod init. Where does v2 come in?

Reply:
That is the path for the root module.  If this works, it's probably not worth changing.  It would only really matter if you were trying to import it from elsewhere.